crates/sncast/src/response/errors.rs

```diff
@@ -73,14 +75,14 @@ pub enum SNCastStarknetError {
     ClassAlreadyDeclared,
     #[error("Invalid transaction nonce")]
     InvalidTransactionNonce,
-    #[error("Max fee is smaller than the minimal transaction cost")]
-    InsufficientMaxFee,
+    #[error("The transaction's resources don't cover validation or the minimal transaction fee")]
+    InsufficientResourcesForValidate,
```

Comment:
So there is no error for insufficient max fee now? Is it some kind of resources bounds error now?

Reply:
Yes, when resource bounds are too low to validate tx, see [spec](https://github.com/starkware-libs/starknet-specs/blob/d70a2e57c9a66db1bbf86c388f38d6295a6a2a75/api/starknet_write_api.json#L42).